src/MobxPromise.ts

```diff
@@ -43,6 +43,8 @@ export type MobxPromiseInputParams<R> = {
      * It will not be called for out-of-date promises.
```

Comment:
@onursumer this is the only file that need be reviewed.  the rest are generated by build.